### DISCUSSION TITLE
Migrate without running unapplied migrations

### DISCUSSION THREAD

**Discussion by chris-chr** [prisma/prisma#7350] [Q&A] [answered]
Hello all,
I would like to know if there is any way to run `prisma migrate dev --create-only` without applying previous migrations. 
Basically all I need is a way to get the differences between `schema.prisma` and the current database schema.

**Comment by ryands17** [comment, marked as answer]
@chris-chr 👋 

I don't think this is possible currently. It would be great if you could create a feature request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) with your use case so that we can look into this :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
